.github/workflows/test.yaml

```diff
@@ -16,6 +16,10 @@ on:
         required: false
         type: string
         default: "gem install semaphore_test_boosters && rspec_booster --job $CI_NODE_INDEX/$CI_NODE_TOTAL"
+      setup_db_cmd:
```

Comment:
dynamic variable created with a default value that most projects use. This supports iiif_print repo because it'll need a different setup command.